inventory_data.go

```diff
@@ -67,6 +67,20 @@ func listRunnable(dpath string) ([]string, error) {
 	return runnable, nil
 }
 
+func (id *InventoryDataRunner) AddDeviceType(device string, inv InventoryData) error {
```

Comment:
This is a bit tricky. I wanted to add a device type only it this is not there already. Maybe instead we should replace existing one with the arbitrary one you are adding here.